pkgs/development/python-modules/matplotlib/default.nix

```diff
@@ -116,11 +119,14 @@ buildPythonPackage rec {
     '';
 
   nativeBuildInputs = [
+    certifi
+    numpy
+    oldest-supported-numpy
```

Comment:
Don't see this one: https://github.com/matplotlib/matplotlib/blob/main/pyproject.toml

Reply:
It won't be needed in 3.8.0 once it's released: https://github.com/matplotlib/matplotlib/commit/8c7a5829e77695c1f80dc176768ddc4bcc092064

Reply:
Can you leave a comment in our code then?

Reply:
```suggestion
    oldest-supported-numpy # replace with `numpy` for 3.8.0
```

Reply:
Good idea! I did that (just tweaked the comment to say remove since numpy is already in nativeBuildInputs).

Reply:
Thanks